docs/release_notes/release_notes-0.14.0.rst

```diff
@@ -4,7 +4,7 @@ Key Features and Updates
 ------------------------
 
 * Stability and Bugfixes
-  *
+  * Allow an empty parse_date list in `read_csv_glob` (f9ff6ac)
```

Comment:
```suggestion
  * FIX-#4056: Allow an empty parse_date list in `read_csv_glob` (#4074)
```

Reply:
I fixed this in the commit that merged master back in.